crates/nu-cli/src/completions/operator_completions.rs

```diff
@@ -32,13 +34,10 @@ impl Completer for OperatorCompletion {
     ) -> Vec<SemanticSuggestion> {
         //Check if int, float, or string
         let partial = std::str::from_utf8(working_set.get_span_contents(span)).unwrap_or("");
-        match &self.previous_expr.expr {
-            Expr::BinaryOp(x, _, _) => match x.expr {
```

Comment:
I think removing this explicit match on `BinaryOp` here increased the false positive rate.

e.g. in argument positions the operator completions are not valid
```
foo 1 <tab>
print "bla" <tab>
```

I gave the previous version a test drive and I think it wasn't suggesting operators and now it does.